docs/posts/into-snowflake/index.qmd

```diff
@@ -0,0 +1,143 @@
+---
+title: "Snow IO: loading data from other DBs into Snowflake"
+author: "Phillip Cloud"
+error: false
+date: "2024-02-23"
+categories:
+  - blog
+  - snowflake
+  - io
+  - productivity
+---
+
+## Recap
+
+We've [blogged about Snowflake IO before](../snowflake-io/index.qmd), in the
+context of getting local files into Snowflake as fast as possible.
+
+In this post, we'll show how to insert query results from another system into
+Snowflake, using Ibis.
+
+## Setup
+
+### Connect to your non-Snowflake system
+
+We'll connect to a postgres database running locally in a container. You
+should be able to swap in your own connection details as needed.
+
+```{python}
+from ibis.interactive import *  # <1>
+
+pg_con = ibis.connect("postgres://postgres:postgres@localhost/postgres")
+```
+
+1. Import Ibis for maximum productivity in interactive analysis.
+
+We'll use a test dataset that contains some baseball batting statistics.
+
+Ibis provides that example data, so we can dump that into postgres.
+
+
+```{python}
+pg_batting = pg_con.create_table(
+    "batting",
+    ibis.examples.Batting.fetch().to_pandas(),  # <1>
```

Comment:
why pandas instead of pyarrow?

Reply:
psycopg2 doesn't seem to be buying what we're selling with `to_pyarrow()` :)

I'll open an issue about it.